enclave_mgmt/models.py

```diff
@@ -0,0 +1,223 @@
+from pydantic import BaseModel, Extra, validator
+from typing import List, Union
+from uuid import UUID
+from datetime import datetime
+from typing import Literal
+from math import isnan
+
+
+class Demographic(BaseModel):
```

Comment:
I think this model is dead code since we removed the corresponding csv in [this PR](https://github.com/openstax/raise-enclave/pull/9).